spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -105,7 +105,7 @@
 
   describe 'GET #show' do
     context 'When fetching an inexistent book' do
-      subject { { error: "Book doesn't found" } }
+      subject { { error: "Couldn't find Book with 'id'=1" } }
```

Comment:
Remember to name your subjects, it's best if you can name it as :error_response, or just avoid it entirely, you can't guarantee the error message will be the same when I18n is implemented.